src/cmd/package.go

```diff
@@ -46,8 +45,8 @@ var packageCreateCmd = &cobra.Command{
 			baseDir = args[0]
 		}
 
-		if zarfImageCache != config.ZarfDefaultImageCachePath && cachePathClean(zarfImageCache) {
-			config.SetImageCachePath(zarfImageCache)
+		if isCachePathClean(config.CreateOptions.CachePath) {
```

Comment:
As a general convention, if it's a public variable already in config, we don't pass those around, but instead use the variable directly.

Reply:
Refactored this